vignettes/pwc_survival.Rmd

```diff
@@ -0,0 +1,138 @@
+---
+title: "Piecewise constant hazards overall survival calculation"
+author: "Daniel Sabanés Bové"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+format:
+  html:
+    html-math-method: mathjax
+    include-in-header:
+      - text: |
+          <script>
+          window.MathJax = {
+            tex: {
+              tags: 'ams'
+            }
+          };
+          </script>
+bibliography: references.bib
+vignette: >
+  %\VignetteIndexEntry{Piecewise constant hazards overall survival calculation}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+## Introduction
+
+In this vignette we derive the explicit formula for the overall survival under
+piecewise constant hazards. This is implemented in the function `PWCsurvOS()`.
+
+So we are in the situation where the hazards for the transitions are piecewise constant,
+i.e. all three hazard functions $\lambda_{01}(t)$ (stable to progression), $\lambda_{02}(t)$ 
+(stable to death) and $\lambda_{12}(t)$ (progression to death) are step functions.
+Say the start points of the constant hazard pieces are $0 \equiv t_{01}^{(1)} < \dotsb < t_{01}^{(k_{01})}$, $k_{01} \geq 1$, for $\lambda_{01}(t)$, with corresponding constant positive hazards $h_{01}^{(1)}, \dotsc, h_{01}^{(k_{01})}$. Obviously we use here the smallest set of pieces, i.e. neighboring hazards are required
```

Comment:
I would slightly change the sentence structure to make it more clear that we talk about the 01 transition as an example. Like this: Say the start points of the constant hazard pieces for $\lambda_{01}(t)$ are $0 \equiv t_{01}^{(1)} < \dotsb < t_{01}^{(k_{01})}$, $k_{01} \geq 1$, with corresponding ...

Reply:
good idea, thanks!